RuddockWebsite/routes.py

```diff
@@ -0,0 +1,363 @@
+from flask import request, session, g, redirect, url_for, abort, \
+    render_template, flash
+from sqlalchemy import text
+import datetime
+import re
+
+from RuddockWebsite import app, auth
+from RuddockWebsite.constants import *
+from RuddockWebsite.common_helpers import *
+from RuddockWebsite.decorators import *
```

Comment:
We should really avoid using `from _ import *`, since that makes it difficult to figure out where functions are being defined as such. I really like using `from RuddockWebsite import module` and later `module.function()`, but otherwise we can also do `from Ruddockwebsite.decorators import login_required` instead of `import *`.

For constants I prefer `import RuddockWebsite.constants as const` and `const.VALUE` over `import *` also.